ui/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx

```diff
@@ -69,9 +70,7 @@ class ResultsExport extends React.Component<Props, State> {
   }
 
   getTransformerType = () => {
```

Comment:
✏️ You can remove the `{}` and the `return` since this a pretty short predicate statement now